### DISCUSSION TITLE
Increasing the width of a column in a table

### DISCUSSION THREAD

**Discussion by daviwall** [Budibase/budibase#2749] [Help] [answered]
Hello again everyone!

I am trying to increase the width of a column on a table. I managed to figured out which CSS is doing that and was able to set the width I want using browser developer tools.
Now I tried adding this CSS to the "edit custom css" but it does not work. Does anyone know how to do that? maybe there is an easier way?

Thank you.

**Comment by mjashanks** [comment, marked as answer]
Unfortunately, is just not possible at the moment. It's been logged before, so we are aware of this issue.

For now, your options are to 
- Use fewer columns
- Use a `Repeater`, and manually add components inside it. Every component that you add to a repeater will get display once per row returned. You can use bindings to make these components display values of the row.

**Reply by daviwall** [reply]
Hello @mjashanks, thanks for your response.
I tried using a single column and still, the field uses 1/4 of the space (150px) as I can see in the CSS. The information (it's a name) gets cut with "...". 
Could this be some sort of bug?
I will try to use the repeater and will report back once done.

Thanks!

**Reply by mjashanks** [reply]
Thanks, @daviwall - I wouldn't say it's a bug so much as we just need to improve our table component to handle column width, text wrapping etc!

**Reply by mjashanks** [reply]
Closing this as it's covered here: https://github.com/Budibase/budibase/discussions/1694 - I've linked to this issue from there
